src/main/java/com/traveltime/sdk/dto/requests/TimeFilterProtoDistanceRequest.java

```diff
@@ -0,0 +1,86 @@
+package com.traveltime.sdk.dto.requests;
+
+import com.igeolise.traveltime.rabbitmq.requests.RequestsCommon;
+import com.igeolise.traveltime.rabbitmq.requests.TimeFilterFastRequestOuterClass;
+import com.traveltime.sdk.auth.TravelTimeCredentials;
+import com.traveltime.sdk.dto.common.Coordinates;
+import com.traveltime.sdk.dto.responses.errors.TravelTimeError;
+import io.vavr.control.Either;
+import lombok.NonNull;
+import okhttp3.Request;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.RandomAccess;
+
+public class TimeFilterProtoDistanceRequest extends ProtoRequest {
```

Comment:
I'm confused why we need a separate type for this

Reply:
Cause it has its own host, countries and transportation modes